Homework5/Task3.py

```diff
@@ -0,0 +1,27 @@
+n = input("Enter the height of the tree: ")
+
+#    *          *          *
+#   /|\         |         /|\
+#  //|\\                 //|\\
+#    |                  ///|\\\  
+#                          |   
+
+if n.isdigit():
+    n = int(n)
+    if n > 0 and n < 50:
+        for i in range (1, n + 1):
+            space = " " * (n - i)
+            branches = "/" * (i - 1) + "|" + "\\" * (i - 1)
+            if i == 1:
```

Comment:
სწორია როგორც გიწერია, 
უბრალოდ შეგეძლო i == 1 ქესი for-ს გარეთ დაგებეჭდა და i == n ქეისიც for-ს გარეთ გქონოდა
უფრო მარტივი გახდებოდა for-ში ლოგიკის წაკითხვა
თუმცა ასეც სწორად იმუშავებს, ეს უბრალოდ nitpicking